modules/packages/Crypto.chpl

```diff
@@ -134,7 +134,7 @@ module Crypto {
       }
       this.buffDomain = {1..this._len};
       for i in this.buffDomain do {
-        this.buff[i] = s.byte(i);
+        this.buff[i] = s.byte(i-1);
```

Comment:
It would be nice to instead change `buffDomain` to be zero-based, but this change is safer. It's OK with me to leave it how you have it.

Reply:
Ironically, I made that change earlier today while doing a last-minute self-review of the module code, then found that doing `start_test test/library/packages/Crypto` didn't actually do anything on my Mac, so then got nervous that I'd silently break something without knowing it, so dropped it back on the floor.  What's the trick to testing Crypto?

Reply:
To test it on Mac you need to `brew install openssl` and
```bash
export LDFLAGS='-L/usr/local/opt/openssl/lib'
export CPPFLAGS='-I/usr/local/opt/openssl/include'
```

Reply:
After doing that, `start_test test/library/packages/Crypto` still wouldn't work because of the skipif.  Taking it one step deeper and trying `start_test test/library/packages/Crypto/saru/`, I get 

```
Package libcrypto was not found in the pkg-config search path.
Perhaps you should add the directory containing `libcrypto.pc'
to the PKG_CONFIG_PATH environment variable
No package 'libcrypto' found
```

Reply:
I ended up doing the development and testing on linux64 instead.  https://github.com/chapel-lang/chapel/pull/14671/commits/29397f8cf9cda0d399949df367ebdca2e4e79765 has the change.